packages/executor/src/utils/constants.ts

```diff
@@ -3,23 +3,9 @@ export enum ExecutorSelectionStrategy {
   SIMPLE_LOCK = '0x0000000000000000000000000000000000000000',
 }
 
-export const customChains = [
-  {
-    network: 'optimisticGoerli',
-    chainId: 420,
-    urls: {
-      apiURL: 'https://api-goerli-optimism.etherscan.io/api',
-      browserURL: 'https://goerli-optimism.etherscan.io',
-    },
-  },
-  {
-    network: 'arbitrum-goerli',
-    chainId: 421613,
-    urls: {
-      apiURL: 'https://api-goerli.arbiscan.io/api',
-      browserURL: 'https://goerli.arbiscan.io/',
-    },
-  },
-]
+// Array of custom chains for Etherscan verification. Each array element must be in the following
+// format:
+// https://hardhat.org/hardhat-runner/plugins/nomiclabs-hardhat-etherscan#adding-support-for-other-networks
+export const customChains = []
```

Comment:
The `optimisticGoerli` and `arbitrumGoerli` chains are already defined in `chainConfig` (imported from @nomiclabs/hardhat-etherscan), so I removed them from this array.